packages/app-headless-cms/src/modules/pageTemplates/views/list/CloneContentModelDialog.tsx

```diff
@@ -0,0 +1,283 @@
+                                                setLocale(value);
+                                                setGroups(null);
+                                            }}
+                                        >
+                                            <Select
+                                                description={t`Choose a locale into which you wish to clone the model`}
+                                                label={t`Content model locale`}
+                                                options={locales}
+                                            />
+                                        </Bind>
+                                    </Cell>
+                                    <Cell span={12}>
+                                        <Bind
+                                            name={"group"}
+                                            validators={validation.create("required")}
+                                        >
+                                            <Select
+                                                description={t`Choose a content model group`}
+                                                label={t`Content model group`}
+                                                options={groups || []}
+                                            />
+                                        </Bind>
+                                    </Cell>
+                                    <Cell span={12}>
+                                        <Bind name="description">
+                                            {props => (
+                                                <Input
+                                                    {...props}
+                                                    rows={4}
+                                                    maxLength={200}
+                                                    characterCount
+                                                    label={t`Description`}
+                                                    value={contentModel.description}
+                                                />
+                                            )}
+                                        </Bind>
+                                    </Cell>
+                                </Grid>
+                            </UID.DialogContent>
+                            <UID.DialogActions>
+                                <ButtonDefault
+                                    onClick={ev => {
+                                        submit(ev);
+                                    }}
+                                >
+                                    + {t`Clone`}
+                                </ButtonDefault>
+                            </UID.DialogActions>
+                        </>
+                    )}
+                </Form>
+            )}
+        </UID.Dialog>
+    );
+};
+
+export default CloneContentModelDialog;
```

Comment:
Avoid `export default`.

Reply:
@brunozoric It's here because `React.lazy` expects a default export, and we code-split these views.